<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Character Creation</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Race">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Race</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Gender">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Stats</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="int_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="ath">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="cha">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="att">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="dex">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Intelligence</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Athleticism</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Charisma</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Attractiveness</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dexterity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pts">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>140</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Debuffs (more points to spend)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="depression">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Depression</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="anxiety">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Anxiety</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="schizophrenia">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Schizophrenia</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="adhd">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ADHD</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="uwalk">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Unable to walk</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cancer">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pinjury">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Predisposition to injury</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="done">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Country of Origin</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Co">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Religion</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Religion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="F_Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="L_Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
